### PR TITLE
Metadata

### LINKED ISSUE #15: Metadata needs research
As discussed, it would be critical to document the WB's metadata needs to see how we incorporate more advanced querying within the space2stats API.

@bpstewar @andresfchamorro; do you have a timeline for this?

### PR DESCRIPTION
**Pull request by zacharyDez**
### What's changed
- **Metadata additions**: Created a STAC catalog from parquet source file.
- **New metadata files**: Added STAC metadata files, including `catalog.json`, `sources.json`, and `space2stats.json`.
- **Notebook updates**: Added a notebook (`metadata.ipynb`) to handle metadata generation for Space2Stats.

### How to test it

1. Open the `metadata.ipynb` notebook and run the cells to generate the STAC catalog.
2. Validate the metadata files in the `stac` folder for completeness and accuracy.
3. Run `stac-check` on the generated STAC catalog to verify metadata compliance.

### Other Notes

- Closes #15

### CODE REVIEW

**Comment by zacharyDez** [comment]
@andresfchamorro, feel free to update the description or add comments here.

**Review by zacharyDez — COMMENTED** [review]
This is looking great @andresfchamorro! Only change requested is moving the folder to a better location.

I see three points where the metadata could come into play:
1. Letting a user browse the metadata and understand the dataset better - could be accomplished via a copy to the STAC browser. Would require that the copy be update regularly. I tried with the github repo file link directly and got a CORS error. 
2. Retrieving fields - After discussing with @alukach, I'd propose we continue relying on the database directly to retrieve the available list of fields. 
3. Ensure consistency between metadata and data upon ingest (#73) - We should add some checks upon ingestion to avoid any drift between the metadata and underlying data in the database. This would make 2 more sensible.

File: notebooks/METADATA/stac/catalog.json
[the remark this answers is not in the document]
Comment (andresfchamorro) [reply notebooks/METADATA/stac/catalog.json:1]: Agreed that we need to move this! Both of those options sound good to me. Would `space2stats_api/src/space2stats_ingest/metadata` make it easier to run the tests you've added?

[the remark this answers is not in the document]
Comment (zacharyDez) [reply notebooks/METADATA/stac/catalog.json:1]: Moved to `space2stats_ingest`!

**Comment by zacharyDez** [comment]
@andresfchamorro @alukach; just need a ✅ to merge

**Review by alukach — APPROVED** [review]

**Review by andresfchamorro — COMMENTED** [review]
@zacharyDez Did you make these changes manually, or can you update the notebook so that we don't recreate it in the future without these changes?

**Review by andresfchamorro — COMMENTED** [review]
Somehow the content on catalog description and purpose got trimmed too...

**Comment by andresfchamorro** [comment]
ok i think i fixed what I mentioned in the comments so I will merge now :) also copied it to gh-pages so we can view it here: https://radiantearth.github.io/stac-browser/#/external/worldbank.github.io/DECAT_Space2Stats/stac/catalog.json?.language=en

**Pull request merged**